thinkhazard_common/models.py

```diff
@@ -40,6 +40,13 @@ class HazardLevel(Base):
     title = Column(Unicode, nullable=False)
     order = Column(Integer)
 
+    @classmethod
+    def get(cls, mnemonic):
+        with DBSession.no_autoflush:
```

Comment:
What I've understood is that this is need to execute select statements inside a transaction.